src/Web/DOM/NodeType.purs

```diff
@@ -35,6 +35,20 @@ instance boundedEnumNodeType :: BoundedEnum NodeType where
   cardinality = Cardinality 12
   toEnum = toEnumNodeType
   fromEnum = fromEnumNodeType
+ 
+instance showNodeType :: Show NodeType where
```

Comment:
Would you mind getting rid of the alignment, so that there's exactly one space before and after each `=`? I think it would be preferable not to have to adjust every line if we end up adding a new constructor with a sufficiently long name.

Reply:
Done